src/rules/add-metadata-to-token.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @title User metadata for Cumul.io
+ * @overview Add Cumul.io user metaadata to token to be used for Cumul.io dashboard filtering
+ * @gallery true
+ * @category marketplace
+ */
+
+
+function addMetadataToToken(user, context, callback) {
+  const namespace = 'https://cumulio/';
+  user.user_metadata = user.user_metadata || {};
+  user.user_metadata.cumulio = user.user_metadata.cumulio || {};
+  Object.keys(user.user_metadata.cumulio).forEach((k) => {
```

Comment:
This won't throw an error if `user.user_metadata.cumulio` is not an object but you might get some strange output if they mis-keyed that property:

<img width="575" alt="Screen Shot 2021-07-20 at 3 05 36 PM" src="https://user-images.githubusercontent.com/855223/126401608-477bc825-8b2e-45d2-9a5f-33d642420431.png">

Should there be maybe a `typeof` check and maybe exit (but not fail) with a `console.log()` if a value is there but it's not an object?